deepmd/pd/model/descriptor/se_atten.py

```
@@ -0,0 +1,1073 @@
+
+    def get_stats(self) -> dict[str, StatItem]:
+        """Get the statistics of the descriptor."""
+        if self.stats is None:
+            raise RuntimeError(
+                "The statistics of the descriptor has not been computed."
+            )
+        return self.stats
+
+    def reinit_exclude(
+        self,
+        exclude_types: list[tuple[int, int]] = [],
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)